### DISCUSSION TITLE
information about the @system function or some way to get the output of an external command

### DISCUSSION THREAD

**Discussion by mmbarret** [taviso/123elf#128] [Q&A]
Hello, I've trying to find information about how to use this function since once, somewhere I saw that it was used to get stock prices from curl, and I'm very interested in getting values from system commands. Can anybody please point me in the correct direction to find the information? Thank you very much.

**Comment by mmbarret** [comment]
Found it, it was under the Q&A section on the wiki. Either way, any pointers would be much welcomed.

**Comment by taviso** [comment]
There is a `{SYSTEM}` macro command, but there is also a `@SYSTEM()` function - but it's only in git and not in a release yet! I guess I could make a new release if that would be helpful.

You just use it like `@SYSTEM("date")` and the cell is populated with the command output. If the command fails, you get `ERR`. You can do simple string manipulation and so on, like `@VALUE(@SYSTEM("stockprice " & A1))`

You will probably need to write a script that gets the output you want, but I think it's pretty handy.

**Reply by mmbarret** [reply]
Thank you very much for the information.

**Reply by mmbarret** [reply]
yes I found out about the @SYSTEM() function on the commit messages, I though that it would be allready present on the release, thanks a lot.
